_security/configuration/tls.md

```diff
@@ -72,7 +72,7 @@ Name | Description
 `plugins.security.ssl.http.keystore_alias` | Alias name. Optional. Default is the first alias.
 `plugins.security.ssl.http.keystore_password` | Keystore password. Default is `changeit`.
 `plugins.security.ssl.http.truststore_type` | The type of the truststore file, JKS or PKCS12/PFX. Default is JKS.
-`plugins.security.ssl.http.truststore_filepath` | Path to the truststore file, which must be under the `config` directory, specified using a relative path. Required.
+`plugins.security.ssl.http.transport.truststore_filepath` | Path to the truststore file, which must be under the `config` directory, specified using a relative path. Required.
```

Comment:
This is the transport truststore filepath, there is also an HTTP equivalent: https://github.com/opensearch-project/security/blob/26b224fa1e2986431f22cf865592c4ce6fa6f49c/src/main/java/org/opensearch/security/ssl/util/SSLConfigConstants.java#L39

You may want to be specific with the description. Otherwise looks good.